docs/standard/base-types/how-to-verify-that-strings-are-in-valid-email-format.md

```diff
@@ -59,7 +59,7 @@ The following example uses a regular expression to verify that a string is in va
 |`(?<=[0-9a-z])`|Continue the match if the character that precedes the @ character is A through Z, a through z, or 0 through 9. The `(?<=[0-9a-z])` construct defines a zero-width positive lookbehind assertion.|  
 |`(?(\[)`|Check whether the character that follows @ is an opening bracket.|  
 |`(\[(\d{1,3}\.){3}\d{1,3}\])`|If it is an opening bracket, match the opening bracket followed by an IP address (four sets of one to three digits, with each set separated by a period) and a closing bracket.|  
-|<code>&#124;(([0-9a-z][-0-9a-z]*[0-9a-z]*\.)+</code>|If the character that follows @ is not an opening bracket, match one alphanumeric character with a value of A-Z, a-z, or 0-9, followed by zero or more occurrences of a hyphen, followed by zero or one alphanumeric character with a value of A-Z, a-z, or 0-9, followed by a period. This pattern can be repeated one or more times, and must be followed by the top-level domain name.|  
+|<code>&#124;(([0-9a-z][-0-9a-z]\*[0-9a-z]\*\\.)+</code>|If the character that follows @ is not an opening bracket, match one alphanumeric character with a value of A-Z, a-z, or 0-9, followed by zero or more occurrences of a hyphen, followed by zero or one alphanumeric character with a value of A-Z, a-z, or 0-9, followed by a period. This pattern can be repeated one or more times, and must be followed by the top-level domain name.|  
```

Comment:
This is adding an unexpected \ character. Here's how your change is rendering:

![image](https://user-images.githubusercontent.com/12971179/65363221-8a7a0600-dbbf-11e9-9e38-332eb14b1339.png)


Maybe like this?

```suggestion
|<code>&#124;(([0-9a-z][-0-9a-z]\*[0-9a-z]\*\.)+</code>|If the character that follows @ is not an opening bracket, match one alphanumeric character with a value of A-Z, a-z, or 0-9, followed by zero or more occurrences of a hyphen, followed by zero or one alphanumeric character with a value of A-Z, a-z, or 0-9, followed by a period. This pattern can be repeated one or more times, and must be followed by the top-level domain name.|  
```

Reply:
I think the baskslash before the `.` is expected. Because `.` in regex means to match anything, it should be escaped here to match the `.` itself.

Reply:
Oh sorry. I've noticed that's how it is in the code.
So can you fix this part where the \ is missing?

![image](https://user-images.githubusercontent.com/12971179/65366482-6c210400-dbd9-11e9-95c4-d4dc54327aa2.png)

Reply:
I'm noticing there are other parts of the regex also missing in the sentence where the whole pattern is shown. Let me know if you'd like me to take care of that on a separate PR.

Reply:
@mairaw, In addition to the sentence where the whole pattern is shown, there were also problems with some table rows.
I fixed those. Not sure if it will look correct now as I don't have access to a live website to see my changes.

Reply:
I'll double check. Thanks!